lib/kube/proxy/forwarder.go

```diff
@@ -864,9 +862,9 @@ func (f *Forwarder) join(ctx *authContext, w http.ResponseWriter, req *http.Requ
 		party := newParty(*ctx, stream.Mode, client)
 		go func() {
 			<-stream.Done()
-			session.mu.Lock()
-			defer session.mu.Unlock()
-			session.leave(party.ID)
+			if err := session.leave(party.ID); err != nil {
+				f.log.Debugf("Participant %q was unable to leave session %s: %v", party.ID, session.id)
```

Comment:
I think this is missing the error. There's 3 format verbs and 2 additional arguments.